### ISSUE TITLE
Adding CRT disabled 

### ISSUE THREAD

**Issue by Nerbderb** [PGgamer2/balatro-mobile-builder#1]
Thanks for the update. Wanted to see if you are able to add the CRT disable setting during install like the original one.

**Comment by PGgamer2** [comment]
I'll add the patch in the next update.
Does your phone have graphical bugs because of it?

**Comment by kimbernator** [comment]
Pixel devices have this issue. I see it on mine, exactly as this screenshot shows: https://github.com/blake502/balatro-mobile-maker/issues/5

**Comment by PGgamer2** [comment]
LOL, I thought I had fixed that problem with [this patch](https://github.com/PGgamer2/balatro-mobile-builder/blob/85098954ecf78ada57b638b3ee2f6114fcd86c99/balatro-mobile-builder/Resources/patches/shaderfixes_crt.patch). I'm adding it back, but the issue will stay open while I investigate further on how to fix that permanently.

**Comment by Nerbderb** [comment]
I only used your latest 0.9.2 and had the issue with the half screen being black. Not sure if it was present in your releases before.

**Comment by PGgamer2** [comment]
Yeah it was already present. I assumed I had fixed that because I don't have a Pixel device and thus no way to properly test the new code.

**Comment by PGgamer2** [comment]
Try the [latest version](https://github.com/PGgamer2/balatro-mobile-builder/releases/tag/beta-0.9.3).

**Comment by Nerbderb** [comment]
Lastest works thanks for the quick turnaround
